alphagov/paas-cf
# [#122189093] Add collectd to bosh and concourse

## What

[Configure Collectd on Concourse and Bosh VMs](https://www.pivotaltracker.com/n/projects/1275640/stories/122189093)

## How to review

Deploy against existing or new environments. 
* Start with bootstrap concourse to update your deployer concourse. 
* Then watch deployer update bosh. 
* After deployment, observe concourse and bosh metrics in graphite under `collectd.<bosh/concourse>.collectd_0.<metrics>`. 
* Once [the pull request on paas-collectd-boshrelease](https://github.com/alphagov/paas-collectd-boshrelease/pull/2) has been merged, @mtekel or @HenryTK will upload the release to github and the FIXME commit can be removed.
* remove release 0.4 from your bosh and deploy again to test that the new release works (the URL is correct, hash matches etc.)
* after depoying to prod, sync master of https://github.com/alphagov/paas-collectd-boshrelease with upstream (= rollback of one PR merge we did on to master), as we are deploying from tarballs now and our main branch is now gds_master

## Who can review

not @mtekel or @HenryTK
